src/v2/providers/pubsub.ts

```diff
@@ -4,20 +4,25 @@ import { CloudEvent, CloudFunction } from '../core';
 import * as options from '../options';
 
 /**
- * A PubSub Topic is:
+ * Google Cloud's Pub/Sub is a globally distributed message bus that automatically scales as you need it.
+ * You can create a function ({@link onMessagePublished}) that handles Pub/Sub events by using functions.pubsub.
+ *
+ * You can trigger a function whenever a new Pub/Sub message is sent to a specific topic.
```

Comment:
Nit: 'This functions triggers whenever a new Pub/Sub...`

Reply:
Makes sense, updated!